Reactions/ReactorArkode.H

```diff
@@ -49,7 +49,11 @@ class ReactorArkode : public ReactorBase::Register<ReactorArkode>
 public:
   static std::string identifier() { return "ReactorArkode"; }
 
+#ifdef AMREX_USE_GPU
```

Comment:
This was verified to not affect the array4 `react()`?

Reply:
Yes, for CPU there doesnt seem to be any appreciable difference in performance.